tests/integration/ceph_upgrade_test.go

```diff
@@ -364,7 +364,8 @@ func (s *UpgradeSuite) verifyRookUpgrade(numOSDs int) {
 func (s *UpgradeSuite) waitForUpgradedDaemons(previousVersion, versionLabel string, numOSDs int, waitForMDS bool) {
 	// wait for the mon(s) to be updated
 	monsNotOldVersion := fmt.Sprintf("app=rook-ceph-mon,%s!=%s", versionLabel, previousVersion)
-	err := s.k8sh.WaitForDeploymentCount(monsNotOldVersion, s.namespace, s.settings.Mons)
+	// extend the timeout for this check because it can take 6+ minutes to pull ceph-ci images
+	err := s.k8sh.WaitForDeploymentCountWithRetries(monsNotOldVersion, s.namespace, s.settings.Mons, utils.RetryLoop+120)
```

Comment:
retry sleep is 5 seconds, so 120 is 10 extra minutes. That's quite a bit, but it could be what's needed to stabilize CI if/when ceph-ci repo is particularly slow.

The detect version job timeout is 15 minutes, and the RetryLoop is 55 by default, which is slightly under 5 minutes. The total timeout waiting on mons is now just below 15 minutes.

Reply:
How long total does this make the retry loop? I don't recall the default. For simplicity, perhaps best just to use a const value such as 150 or 200 instead of adding the `utils.RetryLoop`? Or else a comment for what the total value would be.